scripts/india_plfs/daily_wage_data/README.md

```diff
@@ -0,0 +1,82 @@
+# Periodic Labour Force Survey (PLFS) -  Average daily wage earnings
+
+
+## About the Dataset
+Average wage earnings (Rs. 0.00) per day from casual labour work other than public works in CWS for each State/ UT.
+
+### Download Data
+
+#### PLES July 2018 - June 2019
+Wage data from PLES July 2018 - June 2019 is available as [XLSX](http://mospi.nic.in/sites/default/files/reports_and_publication/PLFS_2018_19_Anual/Table_43.xlsx).
+
+#### PLES July 2017 - June 2018
+Wage data from PLES July 2017 - June 2018 is available as embedded tables inside the  [PDF](http://mospi.nic.in/sites/default/files/publication_reports/Annual%20Report%2C%20PLFS%202017-18_31052019.pdf). Table format is the same. Extracted as XLSX using [tabula](https://github.com/tabulapdf/tabula) by running the command
+
+```
+java -jar tabula-java.jar  -a 40.506,97.049,521.453,782.161 -p 404 "$1" 
```

Comment:
for my understanding, what does -a argument do?

Reply:
`-a/--area = Portion of the page to analyze.`